packages/frontend/core/src/features/auth/components/FormField.tsx

```diff
@@ -0,0 +1,12 @@
+import { TextInput, TextInputProps } from "react-native";
```

Comment:
[author's note] this file complains about not having React Native as a dependency... how do we not have RN as a dependency already???

Reply:
ive seen some issues around this where even if react-native is a peerdependency the ts config still acts as if it cant find it - im pretty sure adding the dependency solves the issue but might be worth looking at a different fix

Reply:
Got it fixed by adding RN as a peer dep 🙏